docs/UserGuide.md

```diff
@@ -139,14 +139,13 @@ Examples:
 
 ### Filtering persons: `find`
 
-* Filter persons based on specific criteria within their records.
+Filter persons based on specific criteria within their records.
 
 Format: `find PREFIX/KEYWORD`
 
 * This command searches for persons using a specific aspect of their details, as specified by the prefix.
 * The search will return any result that contains the keyword as a substring under the indicated prefix. e.g. `find e/hans` will find any person that contains `hans` in their email.
 * The search is case-insensitive. e.g. `hans` will match `Hans`.
-* prefixes that are supported includes : n/, p/, e/, a/, m/ and t/.
 * Only one prefix can be used for filtering at a time.
```

Comment:
maybe you can add a note under this section to tell users how to 'undo' a find command and link it to the `list` command since it is a common confusion?